packages/tools/devtools/devtools-test-app/src/FluidObject.ts

```diff
@@ -175,43 +175,66 @@ export class AppData extends DataObject {
 	}
 
 	private populateSharedTree(sharedTree: ITree): void {
-		// Set up SharedTree for visualization
-		const builder = new SchemaFactory("DefaultVisualizer_SharedTree_Test");
-
-		// TODO: Maybe include example handle
+		const builder = new SchemaFactory("TodoList_Schema");
```

Comment:
I really like this new sample schema! Very easy to reason about, and tests most of our features!

Reply:
Thanks! Glad that this is much semantic than the previous example 😄 .